### PR TITLE
Add GL_KHR_non_const_texture_offset

### PR DESCRIPTION
**Pull request by rg3igalia**

### CODE REVIEW

File: extensions/khr/GL_KHR_non_const_texture_offset.txt [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+
+      #define GL_KHR_non_const_texture_offset        1
+
+
+Modifications to Chapter 8 of the OpenGL Shading Language Specification
+(Built-in Functions)
+
+    Change to section 8.9.2, Texel Lookup Functions
+
+    In the table row corresponding to the textureOffset functions, remove the
```
Comment (lexaknyazev) [inline extensions/khr/GL_KHR_non_const_texture_offset.txt:66]: nit: use `texture*Offset` to include all variants.

Comment (rg3igalia) [reply extensions/khr/GL_KHR_non_const_texture_offset.txt:66]: I can do that, no problem, but it's worth noting all other variants (texelFetchOffset, textureProjOffset, textureLodOffset, textureProjLodOffset, textureGradOffset and textureProjGradOffset) merely reference textureOffset, so the sentence to be removed only appears once.

Comment (lexaknyazev) [reply extensions/khr/GL_KHR_non_const_texture_offset.txt:66]: Ack, the current language is fine.

**Comment by rg3igalia** [comment]
Closing in favor of https://github.com/KhronosGroup/GLSL/pull/264

**Pull request closed without merging**